source/delegate-factory/truffle-config.js

```diff
@@ -1,4 +1,8 @@
+require('dotenv').config()
+const HDWalletProvider = require('truffle-hdwallet-provider')
+
 module.exports = {
+  // contracts_directory: './flatten',
```

Comment:
put in a node like // UNCOMMENT ONLY WHEN DOING A DEPLOYMENT.

Reply:
I still think we can keep this and have migrations ignored for tests.

Reply:
No it still wouldn't work because `truffle test` uses `contracts_directory:` to locate the contracts for tests.

Reply:
Thats not a migration-specific configuration